modules/nw-networkpolicy-about.adoc

```diff
@@ -94,7 +117,7 @@ spec:
 
 For example, for the `NetworkPolicy` objects defined in previous samples, you
 can define both `allow-same-namespace` and `allow-http-and-https` policies
-within the same project. Thus allowing the pods with the label `role=frontend`,
+within the same project. Thus allowing the Pods with the label `role=frontend`,
```

Comment:
"Thus allowing the Pods with the label `role=frontend`,"   This clears up a lot. It should appear sooner.